test/elastic_record/log_subscriber_test.rb

```diff
@@ -16,7 +16,7 @@ def setup
   # end
 
   def test_request_notification
-    FakeWeb.register_uri(:any, %r[/test], status: ["200", "OK"], body: Oj.dump('the' => 'response'))
+    stub_request(:any, "#{Widget.elastic_connection.servers.first}/test").to_return(status: 200, body: Oj.dump('the' => 'response'))
```

Comment:
Huh. The webmock documentation led me to believe that the hostname was mandatory. I will fix.